packages/components/e2e/component-objects/src/test/java/org/talend/component/StorybookTest.java

```diff
@@ -1,44 +1,41 @@
 package org.talend.component;
 
+import org.apache.commons.lang3.StringUtils;
 import org.apache.http.client.utils.URIBuilder;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.support.ui.WebDriverWait;
-
-import java.net.URL;
-import java.net.URLEncoder;
-
-import static org.openqa.selenium.support.ui.ExpectedConditions.elementToBeClickable;
+import org.talend.config.WebDriverTest;
+import org.talend.config.WebDriverTestFactory;
 
 public class StorybookTest {
 
     private static final String ACTION_LOGGER_CONSOLE_SELECTOR = ".horizontal.Pane2 > div > div > div:last-child";
 
-    private static final String STORY_CATEGORY_SELECTOR = "div[data-name='%s']";
-
-    private static final String STORY_MENU_SELECTOR = "a[data-name='%s']";
-
     private static final String DEFAULT_STORY_NAME = "default";
 
     private static final String STORYBOOK_HOST = "localhost";
 
     private static final int STORYBOOK_PORT = 6006;
 
+    private static final String STORYBOOK_CONTEXT = "";
+
+    protected static WebDriverTest webDriverTestConfiguration;
+
     protected static WebDriver driver;
 
     @BeforeClass
-    public static void before() {
-        driver = new ChromeDriver();
-        driver.manage().window().maximize();
+    public static void before() throws Exception {
+        webDriverTestConfiguration = new WebDriverTestFactory().getWebDriverTestConfiguration();
+        webDriverTestConfiguration.setUp();
+        driver = webDriverTestConfiguration.driver;
+        System.out.println("WebDriver is running: " + driver);
```

Comment:
Why didn't you used the `Logger` used in other classes ?

Reply:
👌